packages/expo-image-loader/android/build.gradle

```
@@ -0,0 +1,86 @@
+buildscript {
+  repositories {
+    google()
+    jcenter()
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath 'com.android.tools.build:gradle:3.5.1'
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.41"
+  }
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.0'
+
+def safeExtGet(prop, fallback) {
+  rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
+// Upload android library to maven with javadoc and android sources
+configurations {
+  deployerJars
+}
+
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+  classifier = 'sources'
+  from android.sourceSets.main.java.srcDirs
+}
+
+// Put the androidSources and javadoc to the artifacts
+artifacts {
+  archives androidSourcesJar
+}
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      configuration = configurations.deployerJars
+      repository(url: mavenLocal().url)
+    }
+  }
+}
+
+android {
+  compileSdkVersion safeExtGet("compileSdkVersion", 28)
+
+  defaultConfig {
+    minSdkVersion safeExtGet("minSdkVersion", 21)
+    targetSdkVersion safeExtGet("targetSdkVersion", 28)
+    versionCode 1
+    versionName "1.0.0"
+  }
+  lintOptions {
+    abortOnError false
+  }
+  compileOptions {
+    sourceCompatibility = '1.8'
+    targetCompatibility = '1.8'
+  }
+}
+
+repositories {
+  mavenCentral()
+}
+
+if (new File(rootProject.projectDir.parentFile, 'package.json').exists()) {
+  apply from: project(":unimodules-core").file("../unimodules-core.gradle")
+} else {
+  throw new GradleException(
+      '\'unimodules-core.gradle\' was not found in the usual Flutter or React Native dependency locations. ' +
+          'This package can only be used in such projects. Are you sure you\'ve installed the dependencies properly?')
+}
+
+dependencies {
+  unimodule 'unimodules-core'
+  unimodule 'unimodules-image-loader-interface'
+
+  api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.41"
```

```suggestion
  api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${safeExtGet('kotlinVersion', '1.3.50')}"
```
?

https://github.com/expo/expo/blob/9803cfeb76b9b19d4ca8beb7308278d976f4a9d4/android/build.gradle#L12